doc/README.md

```diff
@@ -4,7 +4,7 @@ Perform actions related to Skynet
 
 ### Synopsis
 
-skynet 1.1.0
+skynet 2.0.1
```

Comment:
Hey, if we're releasing a new major, this is the time to make any breaking changes which we need!
Can we rename `apiKey` to `apiPassword` and remove the ambiguity once and for all?

Reply:
I would actually rename it to skydApiPassword to be more verbose - I think anyone pointing at local skyd would be familiar enough with the prefix